ompi/mca/pml/ob1/pml_ob1_recvreq.c

```diff
@@ -241,6 +242,7 @@ int mca_pml_ob1_recv_request_ack_send_btl(
     des->des_cbfunc = mca_pml_ob1_recv_ctl_completion;
 
     rc = mca_bml_base_send(bml_btl, des, MCA_PML_OB1_HDR_TYPE_ACK);
+
     if( OPAL_LIKELY( rc >= 0 ) ) {
         return OMPI_SUCCESS;
```

Comment:
Do you mean the bytes sent by MPI?  That counter is not updated here, but it probably should be.